hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -115,6 +115,7 @@
       uvm_test_seq: chip_sw_test_base_vseq
       sw_name: flash_ctrl_test
       sw_dir: tests
+      run_opts: ["+sw_test_timeout_ns=10000000"]
```

Comment:
The test passes at 2.1ms (2ms is the default), which takes just 55 seconds to run. It would be better to set the default in `chip_env_cfg` to 5ms instead - otherwise several tests would have to override it.

Reply:
Also set the timeout for the coremark test to 22ms.

Reply:
Updated, thanks